src/Component/Rules/Rules.tsx

```diff
@@ -218,24 +218,32 @@ export const Rules: React.FC<RulesProps> = ({
             editRule(idx);
           }
         }}
+        {...ruleCardProps}
       />
     );
   });
 
-  const defaultActions: ReactNode[] = [
-    <Button
-      onClick={addRule}
-      key={0}
-    >
-      {locale.addRule}
-    </Button>,
-    <Button
-      onClick={classificationClick}
-      key={1}
-    >
-      {locale.classification}
-    </Button>
-  ];
+  const getDefaultActions = (): ReactNode[] => {
```

Comment:
I prefer the approach of just defining a variable instead of a function if no arguments are provided.